CMakeLists.txt

```diff
@@ -89,6 +89,14 @@ option(OGS_USE_EIGEN "Use Eigen linear solver" ON)
 option(OGS_USE_EIGEN_UNSUPPORTED "Use Eigen unsupported modules" ON)
 option(EIGEN_NO_DEBUG "Disables Eigen's assertions" OFF)
 
+# We assume that it's save to use vectorization with Eigen < 3.3 (strictly smaller than 3.3.!).
+# At least we only observed vectorization issues with Eigen 3.3.x.
+# If you want to use Eigen vectorization, make sure that you run all the ctests several times, e.g.:
+# $ ctest --repeat-until-fail 50
+# You might also want to take a look at https://github.com/ufz/ogs/issues/1881.
+# Please see our cmake config files regarding the interplay of this and different Eigen versions!
```

Comment:
Thanks for adopting the comment. But IMHO the last line can be dropped now, since the one reading this comment has already dug into our cmake config, obviously.